src/analyserwaveform.cpp

```diff
@@ -16,6 +16,7 @@ AnalyserWaveform::AnalyserWaveform(ConfigObject<ConfigValue>* pConfig) :
         m_skipProcessing(false),
         m_waveformData(NULL),
         m_waveformSummaryData(NULL),
+        m_stride(),
```

Comment:
I have a slight preference for adding an explicit no-args constructor to WaveformStride rather than relying on value vs. default initialization (because someone might see the no-args constructor call and "optimize it" out since it seems redundant) -- would you be ok with adding one that zeroes all of its fields?